### DISCUSSION TITLE
`gatsby develop` no longer rebuilding on Sass file changes

### DISCUSSION THREAD

**Discussion by DuneArchitect** [gatsbyjs/gatsby#34741] [Help]
Hi everyone,

I recently updated my gatsby version from 3.7.2 to the latest version.
In doing so, I noticed that saving my sass modules no longer triggers a rebuild when server gatsby locally with `gatsby develop`.

I've tracked down that the issue begins before Gatsby v4, and that upgrading to 3.14.x has the same effect.  I'm trying to narrow down exactly which version brings about the breaking change, but it's difficult as selectively upgrading gatsby alone (for example to 3.8.x) causes the build to fail, likely due to some dependency mismatches.

Looking through the release notes, I'm wondering if maybe the webpack cache becoming integrated into the development build might have something to do with it?  Either way, get some pointers as to where I may inspect the webpack configuration and the files watched by `gatsby develop` would be super helpful, or if someone has run into this issue before.

**Comment by rjp44** [comment]
Have a look at https://github.com/gatsbyjs/gatsby/issues/34804#issuecomment-1088447864

I had a similar issue and fixed it by upgrading package versions.

**Comment by DuneArchitect** [comment]
Still having this issue, even for brand new Gatsby installations.  If someone is using CSS modules with the latest version of gatsby sueccessfully, please let me know
